packages/governance/test/unitTests/test-singlePluralityBallotCount.js

```diff
@@ -0,0 +1,276 @@
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { makeStoredPublishKit } from '@agoric/notifier';
+import {
+  eventLoopIteration,
+  makeFakeMarshaller,
+} from '@agoric/notifier/tools/testSupports.js';
+import { makeMockChainStorageRoot } from '@agoric/vats/tools/storage-test-utils.js';
+import { makeHandle } from '@agoric/zoe/src/makeHandle.js';
+import buildManualTimer from '@agoric/zoe/tools/manualTimer.js';
+import { E } from '@endo/eventual-send';
+import {
+  ChoiceMethod,
+  coerceQuestionSpec,
+  ElectionType,
+  QuorumRule,
+} from '../../src/index.js';
+import { makeSinglePluralityVoteCounter } from '../../src/singlePluralityVoteCounter.js';
+
+const SIMPLE_ISSUE = harden({ text: 'Fish or Beef or Pork?' });
+const FISH = harden({ text: 'Fish' });
+const BEEF = harden({ text: 'Beef' });
+const PORK = harden({ text: 'Pork' });
+
+const FAKE_CLOSING_RULE = {
+  timer: buildManualTimer(console.log),
+  deadline: 3n,
+};
+
+const FAKE_COUNTER_INSTANCE = makeHandle('Instance');
+
+function makePublisherFromFakes() {
+  const storageRoot = makeMockChainStorageRoot();
+  const publishKit = makeStoredPublishKit(storageRoot, makeFakeMarshaller());
+  return { publisher: publishKit.publisher, storageRoot };
+}
+
+test('single plurality spoiled', async t => {
```

Comment:
These tests are good. After merging singlePlurality into multiPlurality these should mostly pass as-is, except for the tie breaking.